docs/UserGuide.md

```diff
@@ -110,41 +110,33 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
-### Locating persons by name: `find`
+### Viewing total summary of spendings : `view -s [MONTH]`
 
-Finds persons whose names contain any of the given keywords.
+1. View a summary of all spending <br>
+   * Examples: `view -s`
+   * A summary of total spending: `Total amount spent: $125.30`
+2. View a summary of all spending the specified month
+    * Examples: `view -s 09-2022`
+    * A summary of spendings in September 2022: `Total amount spent in September 2022: $37.70`
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+* View all spending recorded by the user in a month. The month refers to the month that is displayed to the user.
+* The `MONTH` field is optional, if no month is specified, the application displays the current month.
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+### Viewing total summary of income : `view -i [MONTH]`
 
-Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
-
-### Deleting a person : `delete`
-
-Deletes the specified person from the address book.
+1. View a summary of all income <br>
+    * Examples: `view -i`
+    * A summary of total income earned: `Total amount earned: $125.30`
```

Comment:
I'm thinking if we should prefix this with Expected: ... to make it more intuitive to the user?

Reply:
I was wondering about this too, I have changed it to include the Expected outcome.